lib/actions/ssh-keys.ts

```diff
@@ -0,0 +1,141 @@
+/*
+Copyright 2019 Balena Ltd.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { CommandDefinition } from 'capitano';
```

Comment:
For future reference, we are migrating the CLI from capitano to oclif, so a new command could be written using the oclif framework:
* https://github.com/balena-io/balena-cli/blob/master/CONTRIBUTING.md#typescript-vs-coffeescript-and-capitano-vs-oclif
* Existing commands written with oclif:
  * https://github.com/balena-io/balena-cli/blob/master/lib/actions-oclif/version.ts
  * https://github.com/balena-io/balena-cli/blob/master/lib/actions-oclif/env/add.ts
  * https://github.com/balena-io/balena-cli/blob/master/lib/actions-oclif/env/rm.ts

Reply:
Oh, good to know thanks @pdcastro! Since issuing this PR I've actually noticed that there is already a `keys` subcommand so I'll close this! 🤦‍♂